examples/reminders/lib/base/common_ui_components/app_reminder_tile.dart

```diff
@@ -109,8 +109,20 @@ class _AppReminderTileState extends State<AppReminderTile> {
       ));
 
   ActionPane _buildActionPane(BuildContext context) => ActionPane(
+        extentRatio: 0.7,
         motion: const ScrollMotion(),
         children: [
+          SlidableAction(
+            onPressed: (context) =>
+              context.read<ReminderManageBlocType>().events.update(widget
+                  .reminder
+                  .copyWith(complete: !widget.reminder.complete))
+            ,
+            backgroundColor: context.designSystem.colors.activeButtonColor,
```

Comment:
As we discussed the color should change based on the `complete` status

Reply:
This was changed